security/keycloak-multitenant/src/test/java/io/quarkus/ts/openshift/security/keycloak/AbstractSecurityKeycloakOpenShiftIT.java

```diff
@@ -0,0 +1,68 @@
+package io.quarkus.ts.openshift.security.keycloak;
+
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.fabric8.kubernetes.api.model.KubernetesList;
+import io.fabric8.kubernetes.client.utils.Serialization;
+import io.fabric8.openshift.api.model.DeploymentConfig;
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.quarkus.ts.openshift.app.metadata.AppMetadata;
+import io.quarkus.ts.openshift.common.CustomizeApplicationDeployment;
+import io.quarkus.ts.openshift.common.injection.TestResource;
+import io.quarkus.ts.openshift.common.injection.WithName;
+
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Stream;
+
+public abstract class AbstractSecurityKeycloakOpenShiftIT extends AbstractSecurityKeycloakOpenShiftTest {
+
+    static String keycloakUrl;
+
+    // TODO this is pretty ugly, but I'm tired and can't think of a better way at the moment
```

Comment:
Did you by any chance have an epiphany about this yet? :)

Reply:
We could start working on this refactor if required. There is an existing issue for doing so: https://github.com/quarkus-qe/quarkus-openshift-test-suite/issues/127

Maybe something we can ask in our weekly meeting?

Reply:
This is my comment originally, and has been copied many times it seems :-)

Reply:
Right! I don't this to block this MR, so I'm resolving it.